compass/src/main/java/com/marfeel/compass/storage/SessionStorage.kt

```diff
@@ -26,7 +23,9 @@ internal class Memory(private val storage: Storage) {
 		accountId
 
 	fun readSession(): Session =
-		session ?: newSession().also { session = it }
+		storage.readSession() ?: newSession().also {
```

Comment:
storing session data in real storage instead of memory, so we can keep the session open even if they close the app.